docs/mkdocs.yml

```diff
@@ -22,69 +22,79 @@ plugins:
+    - Go: https://pkg.go.dev/github.com/apache/iceberg-go/
   - Integrations:
-    - aws.md
-    - dell.md
+    # Iceberg-maintained integrations come first
+    - Spark:
+      - spark-getting-started.md
+      - spark-configuration.md
+      - spark-ddl.md
+      - spark-procedures.md
+      - spark-queries.md
+      - spark-structured-streaming.md
+      - spark-writes.md
+    - Flink:
+      - flink.md
+      - flink-connector.md
+      - flink-ddl.md
+      - flink-queries.md
+      - flink-writes.md
+      - flink-actions.md
+      - flink-configuration.md
+    - Kafka Connect: kafka-connect.md
+    - Third-party:
+    # Now all the 3rd party integrations
+      - Amazon Athena: https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg.html
+      - Amazon Data Firehose: https://docs.aws.amazon.com/firehose/latest/dev/apache-iceberg-destination.html
+      - Amazon EMR: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg-use-cluster.html
+      - Amazon Redshift: https://docs.aws.amazon.com/redshift/latest/dg/querying-iceberg.html
+      - Apache Doris: https://doris.apache.org/docs/dev/lakehouse/catalogs/iceberg-catalog
+      - Apache Druid: https://druid.apache.org/docs/latest/development/extensions-contrib/iceberg/
+      - Apache Hive: hive.md
+      - ClickHouse: https://clickhouse.com/docs/en/engines/table-engines/integrations/iceberg
+      - Daft: daft.md
+      - Databend: https://docs.databend.com/guides/access-data-lake/iceberg
+      - Dremio: https://docs.dremio.com/data-formats/apache-iceberg/
+      - DuckDB: https://duckdb.org/docs/preview/core_extensions/iceberg/overview
+      - Estuary: https://docs.estuary.dev/reference/Connectors/materialization-connectors/apache-iceberg/
+      - Firebolt: https://docs.firebolt.io/reference-sql/functions-reference/table-valued/read_iceberg
+      - Google BigQuery: https://cloud.google.com/bigquery/docs/iceberg-tables
+      - Impala: https://impala.apache.org/docs/build/html/topics/impala_iceberg.html
+      - Presto: https://prestodb.io/docs/current/connector/iceberg.html
+      - Redpanda: https://docs.redpanda.com/current/manage/iceberg/about-iceberg-topics
+      - RisingWave: risingwave.md
+      - Snowflake: https://docs.snowflake.com/en/user-guide/tables-iceberg
+      - Starrocks: https://docs.starrocks.io/en-us/latest/data_source/catalog/iceberg_catalog
+      - Tinybird: https://www.tinybird.co/docs/forward/get-data-in/table-functions/iceberg
+      - Trino: https://trino.io/docs/current/connector/iceberg.html
+  - Catalogs:
+    - AWS Glue: aws/#glue-catalog
+    - AWS DynamoDB: aws/#dynamodb-catalog
     - jdbc.md
     - nessie.md
-  - API:
-    - java-api-quickstart.md
-    - api.md
     - custom-catalog.md
-  - Javadoc: ../../javadoc/latest/
-  - PyIceberg: https://py.iceberg.apache.org/
-  - IcebergRust: https://rust.iceberg.apache.org/
-  - IcebergGo: https://pkg.go.dev/github.com/apache/iceberg-go/
+  - Storage and Management:
```

Comment:
nit: should this just be `Storage`?  it is not obvious to me where is the `Management` part

Reply:
oh. I actually didn't check out Apache Amoro. I don't think it belongs to `Storage`, which should really be about blob storage like . I don't know what's the best group for it. maybe move it to `Integrations`?

Reply:
As the first step, how about moving everything except modules maintained by iceberg community into `Integrations`?

In the long term, I don't think they should be mixed with documentations for Java libraries.

Reply:
I've moved Amoro into Integrations for now (5293dac5d1d37651ace9633422d90eb85aff44f3); I think perfect will become the enemy of good in this PR otherwise :)

(Probably in the future we might want to consider a more comprehensive list of projects that are offering management&maintenance features, and at that time we could split it out into a new category).

---

@manuzhang we've now got at the top level:

- Concepts
- Libraries
- Integrations
- Catalogs
- Storage

Are you saying Catalogs and Storage should be sub-levels of Integrations? Or that we should have those as categories and just put everything under Integrations regardless of what it does?

Reply:
I also like the separate buckets of catalogs and storage to help navigation